### PR TITLE
DOMA-3480 extend tracking components

### PR DESCRIPTION
**Pull request by sitozzz**
- [x] Update config at values.yaml

### CODE REVIEW

File: apps/condo/domains/common/components/Pickers/DateRangePicker.tsx [hunk 1]
```diff
@@ -20,6 +23,29 @@ const RANGE_PICKER_CSS = css`
...
+
+    const { logEvent, getEventName } = useTracking()
+
+    const eventName = getEventName(TrackingEventType.Daterange)
+
+    const onChangeCallback: RangePickerSharedProps<Dayjs>['onChange'] = (values, formatString) => {
+        if (eventName && formatString.length) {
+            const eventProperties = { component: { value: formatString } }
+
+            const componentId = get(restProps, 'id')
```
Comment (pahaz) [inline apps/condo/domains/common/components/Pickers/DateRangePicker.tsx:36]: it's better to use `props.id`
```suggestion
            const componentId = get(props, 'id')
```

File: apps/condo/domains/common/components/antd/Checkbox.tsx [hunk 2]
```diff
@@ -16,6 +17,11 @@ const Checkbox = (props: CustomCheckboxProps) => {
 
     if (restProps.children && typeof restProps.children === 'string') {
         componentProperties['component'] = { value: restProps.children }
+
+        const componentId = get(restProps, 'id')
```
Comment (pahaz) [inline apps/condo/domains/common/components/antd/Checkbox.tsx:21]:
```suggestion
        const componentId = get(props, 'id')
```

File: apps/condo/domains/common/components/antd/Input.tsx [hunk 3]
```diff
@@ -18,6 +19,11 @@ const Input = (props: CustomInputProps) => {
 
     if (restProps.value) {
         componentProperties['component'] = { value: restProps.value }
+
+        const componentId = get(restProps, 'id')
```
Comment (pahaz) [inline apps/condo/domains/common/components/antd/Input.tsx:23]:
```suggestion
        const componentId = get(props, 'id')
```

File: apps/condo/domains/common/components/antd/Radio.tsx [hunk 4]
```diff
@@ -16,6 +17,11 @@ const Radio = (props: CustomRadioProps) => {
 
     if (restProps.value) {
         componentProperties['component'] = { value: restProps.value }
+
+        const componentId = get(restProps, 'id')
```
Comment (pahaz) [inline apps/condo/domains/common/components/antd/Radio.tsx:21]:
```suggestion
        const componentId = get(props, 'id')
```

File: apps/condo/domains/common/components/antd/Select.tsx [hunk 5]
```diff
@@ -20,11 +22,23 @@ const Select = <T extends string | number | { value: any, label: any }> (props:
...
+            if (isArray(option)) {
+                selectedValue = compact(option.map(opt => get(opt, 'title', false) || get(opt, 'label')))
+            } else {
+                selectedValue = get(option, 'title')
+            }
+
+            if (!isEmpty(selectedValue)) {
                 componentProperties['component'] = { value: selectedValue }
+
+                const componentId = get(restProps, 'id')
```
Comment (pahaz) [inline apps/condo/domains/common/components/antd/Select.tsx:37]:
```suggestion
                const componentId = get(props, 'id')
```

**Review by pahaz — APPROVED** [review]

**Comment by sitozzz** [comment]
https://github.com/open-condo-software/infrastructure-condo/pull/152

**Review by nomerdvadcatpyat — APPROVED** [review]

**Pull request merged**